flink-connectors/flink-connector-kafka/src/test/java/org/apache/flink/connector/kafka/source/reader/KafkaSourceReaderTest.java

```diff
@@ -86,7 +86,7 @@
 public class KafkaSourceReaderTest extends SourceReaderTestBase<KafkaPartitionSplit> {
     private static final String TOPIC = "KafkaSourceReaderTest";
 
-    @BeforeAll
+    @BeforeClass
```

Comment:
Given that SourceReaderTestBase also uses junit5 this doesn't seem correct.

Reply:
Good catch, I'll revert that one for now. But it shows that there is still a lot of work to do